src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -82,7 +82,9 @@ def load_arguments(self, _):
 
     with self.argument_context('container create', arg_group='Network') as c:
         c.argument('network_profile', network_profile_type)
-        c.argument('vnet_name', help='The name of the VNET when creating a new one or referencing an existing one.')
+        c.argument('vnet', help='The name of the VNET when creating a new one or referencing an existing one. Can also reference an existing subnet by ID. This allows using vnets from other resource groups.')
```

Comment:
"Can also reference existing subnet by ID."  This is not true. It would reference an existing Vnet.